src/sets.jl

```diff
@@ -287,6 +287,34 @@ function Base.:(==)(set1::S, set2::S) where S <: Union{PowerCone, DualPowerCone}
     return set1.exponent == set2.exponent
 end
 
+"""
+    NormSpectralCone{T <: Real}(row_dim, column_dim)
+
+The epigraph of the matrix spectral norm (maximum singular value function) ``\\{ (t, X) \\in \\mathbb{R}^{1 + row_dim \\times column_dim} : t \\ge \\sigma_1(X) \\}`` where ``\\sigma_i`` is the ``i``th singular value of the general matrix ``X`` of row dimension `row_dim` and column dimension `column_dim`.
+TODO how to vectorize matrix entries: column or row major?
```

Comment:
I would stack the columns as done in `Base.Array` so that the user can just call `vec`
```julia
julia> A = [1 2; 3 4]
2×2 Array{Int64,2}:
 1  2
 3  4

julia> vec(A)
4-element Array{Int64,1}:
 1
 3
 2
 4
```

Reply:
agreed